### PR TITLE
new module crm_rma_lot_mass_return that adds possibility to ret…

### PR DESCRIPTION
**Pull request by osvalr**
This PR is a splitted of https://github.com/OCA/rma/pull/34 and contains only the crm_rma_lot_mass_return module for better review purposes

### CODE REVIEW

File: crm_rma_lot_mass_return/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'RMA Claims Mass Return by Lot',
+    'version': '1.0',
```
Comment (ajite) [inline crm_rma_lot_mass_return/__openerp__.py]: Please check the guidelines on how to define the module version number.
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

File: crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+# along with this program.  If not, see <http://www.gnu.org/licenses/>. #
+#########################################################################
+
+from openerp import models, fields, api, _
+from openerp.exceptions import except_orm
+
+
+class ReturnedLinesFromSerial(models.TransientModel):
+
+    _name = 'returned_lines_from_serial.wizard'
```
Comment (ajite) [inline crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py]: Should not the _name attribute be as below?

```
_name = 'returned.lines.from.serial.wizard'
```

File: crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+    # Method to get the product id from set
+    @api.model
+    def get_product_id(self, product_set):
+        product_id = False
+        for product in self.prodlot_2_product([product_set]):
+            product_id = product
+        return product_id
+
+    @api.model
+    def create_claim_line(self, claim_id, claim_origine,
```
Comment (ajite) [inline crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py]: s/claim_origine/claim_origin (actually the column should be renamed).

File: crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+        """
+        # If there is a lot number, the product
+        # vendor is searched accurately.
+        inv_obj = self.env['account.invoice']
+        invline_obj = self.env['account.invoice.line']
+        lot_obj = self.env['stock.production.lot']
+        sm_obj = self.env['stock.move']
+
+        user_obj = self.env['res.users']
+        user = user_obj.browse(self._uid)
```
Comment (ajite) [inline crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py]: User object is already available at self.env.user.

File: crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py (hunk 3 above)
Comment (osvalr) [inline crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py]: Hi @ajite 
Why should this column be renamed?

File: crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py (hunk 4 above)
Comment (osvalr) [inline crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py]: @ajite yes, it does, thanks

**Comment by gurneyalex** [comment]
CI is red, setting to Needs fixing.

**Comment by yaniaular** [comment]
@osvalr please, Can you change the tests?. The tests must work starting from purchase order instead of work starting from creating of a picking. The picking must be created from purchase order.

**Comment by osvalr** [comment]
Hi @yaniaular 
Why it should be from purchase order instead of a generated invoice as actually is?

**Comment by osvalr** [comment]
Hi @gurneyalex,

We're working on that

**Comment by yaniaular** [comment]
@osvalr Can you change the tests? before changing something in the code.

**Comment by yaniaular** [comment]
@osvalr Sorry, my bad, rethink:

@osvalr please, Can you change the tests?. The tests must work starting from sale order instead of work starting from creating of a invoice customer. The invoice must be created from sale order.

File: crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py (hunk 3 above)
Comment (ajite) [inline crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py]: Is not the proper spelling "origin"  and not "origine"?

**Comment by osvalr** [comment]
@yaniaular this PR now has a fix for https://github.com/Vauxoo/rma/issues/53, please review ASAP

File: crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+        invoice = self.env['account.invoice']
+        lot = self.env['stock.production.lot']
+
+        prodlot_set_ids = set()
+        lot_set_ids = set()
+        invoice_lot_set_ids = set()
+        lots_lot_set_ids = set()
+
+        if not input_lines:
+            return False, False
```
Comment (yaniaular) [inline crm_rma_lot_mass_return/wizards/returned_lines_from_serial.py]: @osvalr There is a bug here   

```
elements_searched, line_found, lot_lots_ids = \
ValueError: need more than 2 values to unpack
```

You cannot return two values when waiting three values.
Fix it, please.

**Comment by osvalr** [comment]
@yaniaular  @nhomar please review ASAP

**Pull request merged**